Misc/NEWS.d/next/Documentation/2020-02-11-13-22-06.bpo-379070.aaZnkk.rst

```diff
@@ -0,0 +1 @@
+Changed documentation for: Lib/urllib/parse.py at the split and parse functions
```

Comment:
This PR changes the doc-strings, not "the documentation", which is what goes up to docs.python.org.

Also, IMO this doesn't warrant a NEWS entry, so it can and should be removed.

Reply:
@idomic, please remove this file.